testhelpers/helm_helper.go

```diff
@@ -147,6 +147,6 @@ func renderInclude(name string, data interface{}) (string, error) {
 	// require adding the handling of `associated` templates.  A
 	// first run at this generated a stack overflow.  The fake
 	// simply shows what path/name would have been included.
-	name = filepath.Base(name)
-	return name, nil
+	base := filepath.Base(name)
+	return base, nil
```

Comment:
If you're going to change this anyway, it might as well just be `return filepath.Base(name), nil`?

Reply:
True. Will change.

Note, general, changes as I find time in the 1.36 bump.

Reply:
Done with commit 002f7286fb9344c902d70202da9a5a2f56ea39d0